_posts/2023-10-06-k8s-comparison-azure-aws.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+authors: [pieter_vincken]
+title: 'Are all managed Kubernetes clusters created equally?'
+image: /img/2023-10-06-k8s-comparison-azure-aws/header.png
+tags: [cloud, automation, cicd, terraform, iac, azure, aws]
+category: Cloud
+comments: false
+---
+
+> Like stars in the night sky, managed Kubernetes services vary in brilliance; not all shine equally, but the right one can illuminate your journey to cloud-native excellence. - ChatGPT 2023
+
+- [Introduction](#introduction)
+- [The use case](#the-use-case)
+- [Comparison](#comparison)
+- [Architectural components for Kubernetes setup](#architectural-components-for-kubernetes-setup)
+- [How managed is the managed service?](#how-managed-is-the-managed-service)
+- [Out of the box support](#out-of-the-box-support)
+- [Conclusion](#conclusion)
+
+# Introduction
+
+Kubernetes has been the de-facto standard for cloud-native container orchestration for some years now.
+Organizations that rely on a microservices architecture, often rely on Kubernetes to be their platform to deploy to. 
+One of the biggest selling points, especially in recent years, is that Kubernetes can provide a very similar user experience for developers across different on-premise and cloud environments.
+It can provide a common abstraction from the underlying infrastructure through its API. 
+This also allows solutions built on top of Kubernetes to be easily portable across different environments.
+Or at least, that's the promise that is being made by Kubernetes platform teams across the industry.
+
+Running Kubernetes yourself can be very hard to do properly.
+Managing all the moving parts, especially when the workloads are changing frequently can be challenging, to say the least.
+Mix in the usage of more advanced features like a service mesh, custom operators or multi-cloud cluster and it can become a real challenge to maintain.
```

Comment:
Misschien ergens vermelden hier waarom het zo moeilijk is om Kubernetes zelf te managen? mss een eigen ervaring of hypothetisch scenario?